utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java

```diff
@@ -124,14 +125,20 @@ public void insertsIntoTable(String tableName, DataTable data) throws SQLExcepti
 	}
 
 	@Then("^validate that all todos with task \"([^\"]*)\" have value completed \"(\\w+)\", period in ms: \"(\\w+)\"$")
-	public void checksThatAllTodosHaveCompletedVal(String task, Integer val, Integer ms) throws InterruptedException {
+	public void checksThatAllTodosHaveCompletedVal(String task, Integer val, Integer ms) throws InterruptedException, SQLException {
 		Thread.sleep(ms + 1000);
 
 		ResultSet rs;
 		List<Integer> completedAll = new ArrayList<>();
-		String sql = String.format("SELECT completed FROM todo WHERE task like %s", task);
+		String sql = String.format("SELECT completed FROM todo WHERE task like '%s'", task);
+		log.info("SQL **{}**", sql);
 		rs = dbUtils.readSqlOnSampleDb(sql);
-		Assertions.assertThat(rs).isEqualToComparingOnlyGivenFields(val, "completed");
+//		does not work with resultset:
```

Comment:
Yes, we would need `assertj-db` for that. This solution is all right. You can get rid off the commented code.